roles/tpa_single_node/templates/manifests/collector/osv/Deployment.yaml.j2

```diff
@@ -24,60 +25,71 @@ spec:
       annotations:
         configHash/auth: dac9d88322da8b671d9fc4f81500698ea05f108df8784b66db80d4242c9dee06
     spec:
+      volumes:
+        - name: guac-config
+          configMap:
+            name: collectorist-api
+        - name: tls
+          secret:
+            secretName: collector-osv-tls
+        - name: config-auth
+          configMap:
+            name: collector-osv-auth
       containers:
         - name: service
-          image: registry.redhat.io/rhtpa/rhtpa-trustification-service-rhel9@sha256:044b6070e52378a93cbcd65cbd5d10b866f102ee8d6040b92147df7b52202e83
+          image: "{{ tpa_single_node_trustification_image }}"
           imagePullPolicy: IfNotPresent
-          livenessProbe:
-            initialDelaySeconds: 2
-            httpGet:
-              path: /health/live
-              port: 9010
-          readinessProbe:
-            initialDelaySeconds: 2
-            httpGet:
-              path: /health/ready
-              port: 9010
-          command:
-            - /trust
+          command: ["/trust"]
           args:
             - collector
             - osv
             - -p
-            - '8080'
+            - "8080"
             - --auth-configuration
             - /etc/config/auth.yaml
             - --v11y-url
-            - https://v11y-api.trustificcation.svc.cluster.local
+            - https://v11y-api-pod:8080
```

Comment:
Here we need a variable or simply 127.0.0.1 (Which will always work for tpa single node scenario or use a variable with the host ip fact ?
Although I don't think 8080 is the right port because all pods are on the same host's IP.

Reply:
As far as I known the service is running inside pod which is connected with other pods using a network created by podman/kube? I just saw similar approach here: https://github.com/securesign/artifact-signer-ansible/blob/main/roles/tas_single_node/templates/manifests/fulcio/fulcio-server.yaml#L37

The port is the one exposed by the `v11y-api-pod` service. And the name `v11y-api-pod` should be resolved by DNS configured for podman/kube network.

But you are right, we need a variable here.

Reply:
Based on the discussion in the chat, we can make port configurable via a variable later.